src/sentry/runner/__init__.py

```diff
@@ -14,7 +14,7 @@
 if sentry.__build__ is None:
     version_string = sentry.VERSION
 else:
-    version_string = "%s (%s)" % (sentry.VERSION, sentry.__build__[:12])
+    version_string = "{} ({})".format(sentry.VERSION, sentry.__build__[:12])
```

Comment:
That's not an fstring

Reply:
pyupgrade won't do this because it involves more complicated stuff like that attribute access and list slice.

Reply:
It's also kinda unsightly in an f-string. Ideally you give `sentry.__build__[:12]` a name, haha.